R/utils.R

```diff
@@ -71,3 +71,8 @@ dust_header <- function(comment) {
   sprintf("%s Generated by dust (version %s) - do not edit",
           comment, utils::packageVersion("dust"))
 }
+
+
+squote <- function(x) {
+  sprintf("'%s'", x)
+}
```

Comment:
Do you have a dictionary of all of these!?
I tried to make an abind myself recently and it took a while

Reply:
This one I just write out as I need it. `abind` I'd generally use the package, but for our needs in sircovid it was simpler and likely faster to write one out. If it's a complicated utility I usually have a look for the last time I might have written it and copy it over from another package :)